x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/TemplateDeprecationChecker.java

```diff
@@ -122,9 +122,9 @@ private DeprecationIssue checkSourceModeInComponentTemplates(ComponentTemplate t
                     if (sourceMap.containsKey("mode")) {
                         return new DeprecationIssue(
                             DeprecationIssue.Level.CRITICAL,
+                            SourceFieldMapper.DEPRECATION_WARNING_TITLE,
+                            "https://ela.st/migrate-source-mode",
```

Comment:
This for now points at https://github.com/elastic/elasticsearch/pull/117172 but now we can update it when we have the guide, without having as much time pressure. Let me know if you do not agree with this approach.

Reply:
Thanks Mary! We need to regenerate changelogs, because the change now shows up as deprecation instead of a breaking change: https://www.elastic.co/guide/en/elasticsearch/reference/master/migrating-9.0.html#deprecations_90_mapping